### PR TITLE
feat(web): Service web visual changes for Mannauðstorg

### PR DESCRIPTION
**Pull request by RunarVestmann**
# Service web visual changes for Mannauðstorg

## What

* Changed texts and search for the Mannauðstorg service web (/adstod/mannaudstorg)
* Updated the background and footer for the Mannauðstorg service web (/adstod/mannaudstorg)
* Added a new field to SupportQNA's in Contentful called relatedLinks
* Hid the contact us banner for the Mannaudstorg service web and added a new one that's only rendered for the Mannaudstorg page
* Fixed the search input infinite load bug for the service web search input

## Why

* Mannauðstorg wants to change the text, visuals and logic that's on the service web (/adstod/mannaudstorg)

## Screenshots

Here we can see the related links
image

Here's the new look for the homepage of the service web for Mannaudstorg
image

Here's a demonstration for the service web search input fix

https://user-images.githubusercontent.com/43557895/158570145-01f4f6f8-3b2b-459d-9bf1-fcc41a2a38b0.mp4

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/components/Organization/Wrapper/Themes/MannaudstorgTheme/MannaudstorgFooter.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                                underlineVisibility="always"
+                              >
+                                {children}
+                              </Link>
+                            ),
+                          }}
+                        >
+                          <Inline space={1}>
+                            <Text color="white">
+                              {activeLocale === 'is' ? 'Sími:' : 'Telephone:'}
```
Comment (kksteini) [inline apps/web/components/Organization/Wrapper/Themes/MannaudstorgTheme/MannaudstorgFooter.tsx:118]: Not requesting changes necessarily, just food for thought.
What if f.x. Polish gets added to locales in the future?

File: apps/web/screens/ServiceWeb/SubPage/SubPage.tsx [hunk 2]
```diff
@@ -204,10 +224,63 @@ const SubPage: Screen<SubPageProps> = ({
...
+                          )}
+                          {institutionSlugBelongsToMannaudstorg && (
+                            <Box
+                              marginTop={
+                                question.relatedLinks?.length > 0 ? 0 : 4
+                              }
+                            >
+                              <OrganizationContactBanner
+                                organizationLogoUrl={organization.logo?.url}
+                                contactLink="https://gatt.fjs.is/plugins/servlet/desk/portal/11/create/127"
```
Comment (kksteini) [inline apps/web/screens/ServiceWeb/SubPage/SubPage.tsx:271]: Can this not be stored somehow?
If this becomes obsolete a programmer will have to manually update it.

**Review by kksteini — APPROVED** [review]
LGTM

File: apps/web/components/Organization/Wrapper/Themes/MannaudstorgTheme/MannaudstorgFooter.tsx (hunk 1 above)
Comment (RunarVestmann) [reply apps/web/components/Organization/Wrapper/Themes/MannaudstorgTheme/MannaudstorgFooter.tsx:118]: You are correct, this is something on the backlog that we will address at a later point.

File: apps/web/screens/ServiceWeb/SubPage/SubPage.tsx (hunk 2 above)
Comment (RunarVestmann) [reply apps/web/screens/ServiceWeb/SubPage/SubPage.tsx:271]: This link will be different depending on what support qna you are looking at, I know of the technical debt and I will revisit this at a later point when it is known how the links will be implemented.

**Review by siggifv — APPROVED** [review]

**Pull request merged**